pom.xml

```diff
@@ -56,6 +56,11 @@
         <embeddedMongoDbLogDir>${project.build.directory}/embeddedMongoDb/logs</embeddedMongoDbLogDir>
         <!-- Not using default port to reduce chances of shutting down an external MongoDB instance -->
         <embeddedMongoDbPort>27018</embeddedMongoDbPort>
+
+        <!-- Since we require users to import a specific Hibernate Search module explicitly,
+         all integration tests need to know which version that shall be.
+         So the property needs to be defined in the global parent pom -->
+        <hibernate-search.module.slot>5.1</hibernate-search.module.slot>
```

Comment:
Why is it that 5.1 is used now? Is there a specific need?

If we explicitly need 5.1, I think there should be a note in the reference guide regarding the set-up in WildFly, specifically that the HSEARCH 5.1 module ZIP needs to be put into the installation.

This used to be part of the OGM ZIP (just repackaging the original HSEARCH ZIP) for the sake of convinience, but I guess this was removed at some point when we used the WF stock HSEARCH. We also might re-establish that mechanism.